frontend/wallet/src/render/i18n/translations/en.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "id": "account-settings",
+    "defaultMessage": "account settings"
+  },
+  {
+    "id": "cancel",
+    "defaultMessage": "cancel"
+  },
+  {
+    "id": "catching up",
+    "defaultMessage": "catching up"
+  },
+  {
+    "id": "close",
+    "defaultMessage": "close"
+  },
+  {
+    "id": "connecting",
+    "defaultMessage": "connecting"
+  },
+  {
+    "id": "error",
+    "defaultMessage": "error"
+  },
+  {
+    "id": "footer.compress-blockchain",
+    "defaultMessage": "compress the blockchain"
+  },
+  {
+    "id": "listening",
+    "defaultMessage": "listening"
+  },
+  {
+    "id": "memo",
+    "defaultMessage": "memo"
+  },
+  {
+    "id": "recipient",
+    "defaultMessage": "recipient"
+  },
+
+  {
+    "id": "request",
+    "defaultMessage": "request"
+  },
+  {
+    "id": "send",
+    "defaultMessage": "send"
+  },
+  {
+    "id": "sender",
+    "defaultMessage": "sender"
+  },
+  {
+    "id": "settings",
+    "defaultMessage": "settings"
+  },
+  {
+    "id": "sidebar.new-account",
```

Comment:
for components with specific texts, it's best to namespace them accordingly. usually <component or filename.content>